projects/kit/components/stepper/stepper.component.ts

```diff
@@ -56,19 +50,19 @@ export class TuiStepperComponent {
     orientation: TuiOrientation = 'horizontal';
 
     @Input('activeItemIndex')
-    set activeIndex(index: number) {
-        this.activeItemIndex = index;
-        this.scrollIntoView(index);
-    }
+    activeItemIndex = 0;
 
     @Output()
     readonly activeItemIndexChange = new EventEmitter<number>();
 
-    activeItemIndex = 0;
-
     constructor(
         @Inject(ChangeDetectorRef) private readonly changeDetector: ChangeDetectorRef,
-    ) {}
+        @Inject(ElementRef) private readonly elementRef: ElementRef<HTMLElement>,
+        @Inject(TuiScrollService) private readonly scrollService: TuiScrollService,
+        @Inject(TuiResizeService) resize$: Observable<void>,
+    ) {
+        resize$.subscribe(() => this.changeDetector.markForCheck());
```

Comment:
Why do we need it? It's a heavy performance hit.

Reply:
finally remade without hooks, this way seems like performance is ok
take a look plz